docker-compose.yml

```diff
@@ -12,6 +12,7 @@ networks:
         config:
           -
             subnet: 10.0.0.0/24
+    tails-server:
```

Comment:
I think we want the tails server to be on the same network as everything else (ie agency-network). Otherwise agents won't be able to talk to the tails server.